mypy/test/data.py

```diff
@@ -657,17 +656,20 @@ def has_stable_flags(testcase: DataDrivenTestCase) -> bool:
 class DataSuite:
     # option fields - class variables
     files = None  # type: List[str]
-    base_path = '.'
+
+    base_path = test_temp_dir
+
+    # Allow mypyc to update to using a recent version of mypy. See #4779
```

Comment:
I would say "Allow external users of the test code to override the data prefix" or something